internal/runner/rspec.go

```diff
@@ -63,3 +62,22 @@ func (Rspec) discoveryPattern() DiscoveryPattern {
 		ExcludePattern: excludePattern,
 	}
 }
+
+// commandNameAndArgs returns the command name and arguments to run the Rspec tests
+func (Rspec) commandNameAndArgs(testCases []string, testCommand string) (string, []string) {
+	completedTestCommand := ""
+	if strings.Contains(testCommand, "{{testExamples}}") {
+		completedTestCommand = strings.ReplaceAll(testCommand, "{{testExamples}}", strings.Join(testCases, " "))
+	} else {
+		completedTestCommand = testCommand + " " + shellquote.Join(testCases...)
+	}
+
+	testCommandFields, err := shellquote.Split(completedTestCommand)
+
+	if err != nil {
+		fmt.Printf("Couldn't process test command %q: %v", testCommand, err)
+		os.Exit(3)
```

Comment:
We want to rerun rspec tests when having rspec failure (Rspec failure will exit with exit code 1). I'm using error code 3 here so we can differentiate rspec failure and splitter client failure.